user-guide/monitoring.md

```diff
@@ -147,75 +146,76 @@ We have published a [sample dashboard](https://grafana.com/dashboards/10434) you
 
 **Note:** If you deployed the Prometheus Operator via the Helm Chart, a Grafana dashboard is created by default. You can use this dashboard or set `grafana.enabled: false` and follow the instructions below.
 
-To deploy Grafana behind Ambassador: replace `{{AMBASSADOR_IP}}` with the IP address of your Ambassador service, copy the YAML below, and apply it with `kubectl`:
+To deploy Grafana behind Ambassador: replace `{{AMBASSADOR_IP}}` with the IP address or DNS name of your Ambassador service, copy the YAML below, and apply it with `kubectl`:
 
 ```yaml
 ---
-apiVersion: apps/v1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
   name: grafana
-  namespace: default
+  labels:
+    app: grafana
+    component: core
 spec:
   replicas: 1
   selector:
     matchLabels:
       app: grafana
-      componenet: core
+      component: core
   template:
     metadata:
+      creationTimestamp: null
       labels:
         app: grafana
         component: core
+      annotations:
+        sidecar.istio.io/inject: 'false'
     spec:
-      containers:
-      - image: grafana/grafana:6.2.0
-        name: grafana-core
-        imagePullPolicy: IfNotPresent
-        resources:
-          limits:
-            cpu: 100m
-            memory: 100Mi
-          requests:
-            cpu: 100m
-            memory: 100Mi
-        env:
-          - name: GF_SERVER_ROOT_URL
-            value: {{AMBASSADOR_IP}}/grafana
-          - name: GF_AUTH_BASIC_ENABLED
-            value: "true"
-          - name: GF_AUTH_ANONYMOUS_ENABLED
-            value: "false"
-        readinessProbe:
-          httpGet:
-            path: /login
-            port: 3000
-        volumeMounts:
-        - name: grafana-persistent-storage
-          mountPath: /var
       volumes:
-      - name: grafana-persistent-storage
-        emptyDir: {}
-```
-
-Now, create a service and `Mapping` to expose Grafana behind Ambassador:
-
-```yaml
----
-apiVersion: getambassador.io/v1
-kind: Mapping
-metadata: 
-  name: grafana
-  namespace: monitoring
-spec:     
-  prefix: /grafana/
-  service: grafana.monitoring
+        - name: data
+          emptyDir: {}
+      containers:
+        - name: grafana
+          image: 'grafana/grafana:6.4.3'
+          ports:
+            - containerPort: 3000
+              protocol: TCP
+          env:
+            - name: GF_SERVER_ROOT_URL
+              value: {{AMBASSADOR_IP}}/grafana
+            - name: GRAFANA_PORT
+              value: '3000'
+            - name: GF_AUTH_BASIC_ENABLED
+              value: 'false'
+            - name: GF_AUTH_ANONYMOUS_ENABLED
+              value: 'true'
+            - name: GF_AUTH_ANONYMOUS_ORG_ROLE
+              value: Admin
```

Comment:
I was going to say we should change this back so people have to log in but now that I think about it more, that goes against the story of all auth should be handled at the gateway. 🙂

Reply:
No work to be done here just wanted to comment on the good change 😄